CommandHelper/LocalPackages/commands/chars/guardian(MD).yml

```diff
@@ -0,0 +1,22 @@
+class: guardian
```

Comment:
the glass and the file name must match exactly, excluding the file extension .yml